cert.py

```diff
@@ -144,17 +155,32 @@ def _get_http_challenge(ctx, authzr):
     ctx.fail('no acceptable challenge type found; only HTTP01 supported')
 
 
-def _store_webroot_validation(webroot, challb, val):
+def _store_webroot_validation(ctx, remote, webroot, challb, val):
     logger.info('storing validation of %s' % webroot)
-    try:
-        os.makedirs(os.path.join(webroot, challb.URI_ROOT_PATH), 0o755)
-    except OSError as e:
-        if e.errno != errno.EEXIST:
-            raise
-
-    with open(os.path.join(webroot, challb.path.strip('/')), 'wb') as outf:
-        logger.info('storing validation to %s' % outf.name)
-        outf.write(b(val))
+    chall_path = os.path.join(webroot, challb.path.strip('/'))
+    if not remote:
+        try:
+            os.makedirs(os.path.join(webroot, challb.URI_ROOT_PATH), 0o755)
+        except OSError as e:
+            if e.errno != errno.EEXIST:
+                raise
+
+        with open(chall_path, 'wb') as outf:
+            logger.info('storing validation to %s' % outf.name)
+            outf.write(b(val))
+    else:
+        logger.info('storing validation to %s:%s' % (remote[1], chall_path))
+        ssh = pxssh()
+        ssh.force_password = True
```

Comment:
This was just for POC. Temporary workaround for problem with GUI popup dialog when you type wrong password. From official pxssh documentation: "Many X display managers (xdm, gdm, kdm, etc.) will automatically start a GUI agent. You may see a GUI dialog box popup asking for a password during development. You should turn off any key agents during testing. The ‘force_password’ attribute will turn off public key authentication."

I am willing to fix this somehow if we agree to merge this new feature.